### DISCUSSION TITLE
关于我 | return 0;

### DISCUSSION THREAD

**Discussion by HeRaNO** [HeRaNO/HeRaNO.github.io#168] [Archive]
https://herano.github.io/about/ 

 电子科技大学某看起来是一本专业实际上是文科专科软件工程（系统与技术）专业学生，现在大四。将于 2022 年入学哈尔滨工业大学，在计算学部读研。 2021 年上半年在字节跳动实习，负责电商业务后端开发。 没啥爱好，没啥人设，透明属性，音游萌新。 目前在上手 Lutece 的后端。 目测转不了行了，人也越来越现实又无趣了。对不起。

**Comment by junjiezou1** [comment]
佬很厉害呀，膜拜大佬
